ui/widget.js

```diff
@@ -390,6 +409,64 @@ $.Widget.prototype = {
 		return this._setOptions({ disabled: true });
 	},
 
+	_classes: function( options ) {
+		var full = [],
+			that = this,
+			settings = $.extend( {}, {
```

Comment:
The empty object is unnecessary. You're creating a new object on every call anyway.